### PR TITLE
Bump version, add change log, remove history log (deprecated). 

### PR DESCRIPTION
**Pull request by eb-enzo**

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Changelog
+=========
+
+0.0.16 (2017-07-07)
+-------------------
+- Initial release
+
+0.0.17 (2017-11-28)
+-------------------
+Correctly scale the flame graph so it shows all the samples.
```
Comment (nickwilliams-eventbrite) [inline CHANGELOG.txt]: These are in the incorrect order. Newest versions always come first in the file, so 0.0.17 should be above 0.0.16.

File: CHANGELOG.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Changelog
+=========
+
+0.0.16 (2017-07-07)
+-------------------
+- Initial release
```
Comment (ebpitts) [inline CHANGELOG.txt]: That's misleading.

File: CHANGELOG.txt (hunk 1 above)
Comment (nickwilliams-eventbrite) [reply CHANGELOG.txt]: Items under a release heading should each start with a leading hyphen.

File: CHANGELOG.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Changelog
+=========
+
+0.0.16 (2017-07-07)
+-------------------
+- Bug fixes for endpoints that do not perform queries nor have templates.
+
+0.0.17 (2017-11-28)
+-------------------
+- Correctly scale the flame graph so it shows all the samples.
```
Comment (nickwilliams-eventbrite) [inline CHANGELOG.txt]: You did not address all of my comments.

File: CHANGELOG.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+=========
+
+0.0.17 (2017-11-28)
+-------------------
+- Correctly scale the flame graph so it shows all the samples.
+
+0.0.16 (2017-07-07)
+-------------------
+- Bug fixes for endpoints that do not perform queries nor have templates.
+
```
Comment (nickwilliams-eventbrite) [inline CHANGELOG.txt]: You have an extra newline at the bottom here. No biggie, but you should remove it.

**Pull request merged**